datafusion/core/tests/fuzz_cases/join_fuzz.rs

```diff
@@ -383,7 +419,7 @@ impl JoinFuzzTestCase {
 
     /// Perform sort-merge join and hash join on same input
     /// and verify two outputs are equal
-    async fn run_test(&self) {
+    async fn run_test(&self, join_test: &[JoinTest], debug: bool) {
```

Comment:
Since this is already on a struct, another way to pass in the `debug` flag would be as a new field on `JoinFuzzTestCase`


like
```rust
struct JoinFuzzTestCase {
    batch_sizes: &'static [usize],
    input1: Vec<RecordBatch>,
    input2: Vec<RecordBatch>,
    join_type: JoinType,
    join_filter_builder: Option<JoinFilterBuilder>,
    // if debug flag is set the test will save randomly generated inputs and outputs to user folders
    // so it is easy to run debug on top of the failed test data
    debug: bool,
}
```